docs/source/why.rst

```diff
@@ -0,0 +1,187 @@
+Why Dask?
+=========
+
+This document gives high-level motivation on why people choose to adopt Dask.
+
+.. toctree::
+   :maxdepth: 1
+
+Familiar API
+------------
+
+Analysts often use tools like Pandas, Scikit-Learn, Numpy, and the rest of the
+Python ecosystem to analyze data on their personal computer.  They like these
+tools because they are efficient, intuitive, and widely trusted.  However when
+they choose to scale their analyses to larger datasets these tools do no scale
+well, and the analyst is forced to rewrite their computation using a more
```

Comment:
"these tools do not scale well" -> "These tools were not designed for ...", where "..." is something like "larger datasets" or "distributed computing" or something along those lines.

Just to keep things positive :)